### PR TITLE
#22 Added contact me features

### PR DESCRIPTION
**Pull request by colebidex2483**
#22 
Steps to follow.
1. create a form "contact" from the cp
2. create a form field according to the specification provided in the issues(Name,Emal,Message)
 
**For the email**
just run this from the terminal	
php artisan queue:table
php artisan migrate 
php artisan queue:work

### CODE REVIEW

File: app/Jobs/EmailJob.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+namespace App\Jobs;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+
+class EmailJob implements ShouldQueue
```
Comment (davorminchorov) [inline app/Jobs/EmailJob.php:12]: Is this file needed? Looks empty.

File: config/queue.php [hunk 2]
```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    'default' => env('QUEUE_CONNECTION', 'sync'),
+    'default' => env('QUEUE_DRIVER', 'database'),
```
Comment (davorminchorov) [inline config/queue.php:16]: Is this change for Laravel 9 or Laravel 10?

File: database/migrations/2023_03_29_113823_create_jobs_table.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('jobs', function (Blueprint $table) {
```
Comment (davorminchorov) [inline database/migrations/2023_03_29_113823_create_jobs_table.php:16]: You can also generate the failed jobs migration as well.

File: config/queue.php (hunk 2 above)
Comment (davorminchorov) [reply config/queue.php:16]: Also, these types of changes need to be done in the .env file, not directly in the config defaults.

File: app/Jobs/EmailJob.php (hunk 1 above)
Comment (colebidex2483) [reply app/Jobs/EmailJob.php:12]: that's true!!  I will remove it. I thought to have a queue you must implement the shouldQueue interface, which is why I created the file.

File: config/queue.php (hunk 2 above)
Comment (colebidex2483) [reply config/queue.php:16]: Noted will update it

File: database/migrations/2023_03_29_113823_create_jobs_table.php (hunk 3 above)
Comment (colebidex2483) [reply database/migrations/2023_03_29_113823_create_jobs_table.php:16]: it appears, there is already a failed job migration, you can check that up.

File: resources/views/vendor/notifications/email.blade.php [hunk 4]
```diff
@@ -0,0 +1,58 @@
+<x-mail::message>
```
Comment (davorminchorov) [inline resources/views/vendor/notifications/email.blade.php:1]: Is this file used?

Comment (colebidex2483) [reply resources/views/vendor/notifications/email.blade.php:1]: No.

Comment (davorminchorov) [reply resources/views/vendor/notifications/email.blade.php:1]: You can remove it.

**Review by davorminchorov — APPROVED** [review]
